dev_guide/app_tutorials/ruby_on_rails.adoc

```diff
@@ -0,0 +1,357 @@
+  host: <%= ENV["#{db_service}_SERVICE_HOST"] %>
+  port: <%= ENV["#{db_service}_SERVICE_PORT"] %>
+  database: <%= ENV["POSTGRESQL_DATABASE"] %>
+----
+
+For an example of how the final file should look, check out link:https://github.com/openshift/rails-ex[Ruby on Rails example application] link:https://github.com/openshift/rails-ex/blob/master/config/database.yml[config/database.yml].
+
+=== Store your App in Git
+
+OpenShift requires link:http://git-scm.com/[git], if you don't have it installed you will need to install it.
+
+Building an application in OpenShift usually requires that the source code be stored in a link:http://git-scm.com/[git] repository, so you will need to install `git` if you do not already have it.
+
+Make sure you are in your Rails application directory by running `ls -1` command. The output of the command should look like:
+
+----
+$ ls -1
+app
+bin
+config
+config.ru
+db
+Gemfile
+Gemfile.lock
+lib
+log
+public
+Rakefile
+README.rdoc
+test
+tmp
+vendor
+----
+
+Now run these commands in your Rails app directory to initialize and commit your code to git:
+
+----
+$ git init
+$ git add .
+$ git commit -m "initial commit"
+----
+
+Once your application is committed you need to push it to a remote repository. For this you would need a link:https://github.com/join[GitHub account], in which you link:https://help.github.com/articles/creating-a-new-repository/[create a new repository].
+
+Set the remote that points to your `git` repository:
+
+----
+$ git remote add origin git@github.com:<namespace/repository-name>.git
+----
+
+After that, push your application to your remote git repository.
+
+----
+$ git push
+----
+
+== Deploy your application to OpenShift
+
+To deploy your Ruby on Rails application we will need to create a new link:../projects.html[Project] in which our application will live, by typing:
+
+----
+$ oc new-project rails-app --description="My Rails application" --display-name="Rails Application"
+----
+
+After creating the the `rails-app` link:../projects.html[project], you will be automatically switched to the new project.
+
+Deploying your application in OpenShift will be done in three steps:
+
+* Creating database link:../../architecture/core_concepts/pods_and_services.html#services[service] from OpenShift link:../../using_images/db_images/postgresql.html[PostgreSQL image]
+* Creating frontend link:../../architecture/core_concepts/pods_and_services.html#services[service] from OpenShift link:../../using_images/sti_images/ruby.html[Ruby 2.0 builder image] and your Ruby on Rails source code, which we wire with the database service
+* Creating a route for your application.
+
+=== Creating database service
+
+Our Rails application will expect a running database link:../../architecture/core_concepts/pods_and_services.html#services[service] running. For this service we will use link:http://www.postgresql.org/[PostgeSQL] database link:../../using_images/db_images/postgresql.html[image].
```

Comment:
s/Our/Your/
s/will expect/expects/
delete "running" at end of sentence (redundant)
suggest: "This example uses the link...[Postgres] ..." (avoid "we will"; add missing article)